src/extension/commands/debug.ts

```diff
@@ -552,7 +552,7 @@ export class DebugCommands implements IAmDisposable {
 	}
 
 	private async handleCustomEventWithSession(session: DartDebugSessionInformation, e: vs.DebugSessionCustomEvent) {
-		this.vmServices.handleDebugEvent(session, e)
+		this.vmServices.handleDebugEvent(session, e, this.workspaceContext.config.forceFlutterWorkspace)
```

Comment:
I think it might be neater to pass the whole wsContext or config into the constructor (up on line 65) so that it's just generally available to that class.